github_flow.md

```diff
@@ -0,0 +1,54 @@
+# Github flow
+
+## We have agreed, that we will work using Github flow. Simple instructions can be found below.
+
+In short words, we are forking repository from Code for Poznan organisation to our repository, push the changes there and create pull requests from our branches (eg. ``w1stler:sample_branch``) to ``codeforpoznan:master``. 
+
+## Link for reference: https://guides.github.com/introduction/flow/
+
+How to setup your local repository?
+
+1. Fork repository from Code for Poznań
+2. Clone your (forked) repository to local computer
+3. Add upstream remote by executing command ``git remote add upstream git@github.com:CodeForPoznan/project_name.git``
```

Comment:
sometimes You use one ` and sometimes double ` for oneliners. Can we decided and stick to one of them?